osidb/tests/test_embargo.py

```diff
@@ -4,7 +4,7 @@
 
 from osidb.models import Flaw
 
-from .factories import FlawFactory
+from .factories import FlawFactory, AffectFactory
```

Comment:
needs to be in the alphabetic order (isort)

Reply:
it is locally enables, not sure why it fail to fail. Will change thanks.